Kotlin/lib/src/main/java/com/wolt/blurhashkt/BlurHashDecoder.kt

```diff
@@ -21,8 +21,6 @@ object BlurHashDecoder {
         '|', '}', '~'
     )
 
-    private val MAX_SIZE = 20
```

Comment:
unused

Reply:
The value is recommended in the `Readme.md`. It is unused for now, but it should be public and used [here](https://github.com/woltapp/blurhash/blob/6a0abe6e2f57d3960b23d8f0fcece2bc27d09ac9/Kotlin/demo/src/main/java/com/wolt/blurhashapp/MainActivity.kt#L28). If you could update that line, the PR is ready to go. Good stuff!

Reply:
Done!